packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -730,11 +799,50 @@ export class GarbageCollector implements IGarbageCollector {
         // Log pending unreferenced events such as a node being used after inactive. This is done after GC runs and
         // updates its state so that we don't send false positives based on intermediate state. For example, we may get
         // reference to an unreferenced node from another unreferenced node which means the node wasn't revived.
-        await this.logPendingEvents(logger);
+        await this.logUnreferencedEvents(logger);
 
         return gcStats;
     }
 
+    private logSweepEvents(logger: ITelemetryLogger, currentReferenceTimestampMs?: number) {
+        /**
+         * For nodes that are ready to sweep, log an event for now. Until we start running sweep which deletes objects,
+         * this will give us a view into how much deleted content a container has.
+         */
+        if (this.mc.config.getBoolean(disableSweepLogKey) === true
+            || currentReferenceTimestampMs === undefined
+            || this.sweepTimeoutMs === undefined) {
+            return;
+        }
+
+        this.unreferencedNodesState.forEach((nodeStateTracker, nodeId) => {
+            if (nodeStateTracker.state !== UnreferencedState.SweepReady) {
+                return;
+            }
+
+            const nodeType = this.runtime.getNodeType(nodeId);
+            if (nodeType !== GCNodeType.DataStore && nodeType !== GCNodeType.Blob) {
+                return;
+            }
+
+            // Log deleted event for each node only once to reduce noise in telemetry.
+            const uniqueEventId = `Deleted-${nodeId}`;
```

Comment:
We can rename this later with no back-compat thoughts, right?  It's just in-memory for deduping within a session.

Reply:
yes